### ISSUE TITLE
Disable all keyboard functionality in NVDA

### ISSUE THREAD

**Issue by bzkmsjy** [nvaccess/nvda#16582]
I am not completely blind, but my eyes cannot work for long periods. I need the mouse navigation feature of NVDA to read out the content on the screen using the mouse pointer to alleviate my deteriorating vision. However, I do not want to change my keyboard usage habits. I want all the keys on the keyboard to work the same as they did before installing NVDA, such as using the arrow keys to adjust the video progress and volume in web videos. Currently, NVDA does not provide this option, or I have not found it (I do not understand English and have difficulty searching for English content). I hope NVDA can offer an option or shortcut key to disable all keyboard functions. Thank you!

**Comment by Adriani90** [comment]
This happens because NVDA enters browse mode on every website when the page is loaded. You can turn off  browse mode on every page load in NVDA‘s browse mode settings. I am closing this issue as invalid.

**Comment by hwf1324** [comment]
@Adriani90

No, I don't agree with just closing this Issue.
Although this approach does solve one of the issues raised in the problem description.
But this Issue description is disabling NVDA's response to the keyboard, and while this may not meet the needs of most users, it should be up to NV Access to make a decision on whether or not to accept it.

**Comment by CyrilleB79** [comment]
@bzkmsjy could you try to disable "Enable navigation mode on page load" in Browse mode settings? Does this fit your needs?

If not, please explain why and we can reopen this issue. Thanks.

**Comment by josephsl** [comment]
Hi,

More specifically, please tell us if this is for web browsing or to disable NVDA keyboard commands entirely. That way we can figure out if it is doable or not and fits the purpose of screen readers and their keyboad nput mechanisms.

Thanks.
